### PR TITLE
[DO NOT MERGE]CORTX-32188: Fix 42motr-utils and 45motr-rmw common ST failures for New Btree

### PR DESCRIPTION
**Pull request by RadhaGulhane13**
# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

-----
[View rendered README.md](https://github.com/RadhaGulhane13/cortx-motr/blob/rgw_btree_increase_tx/README.md)

### CODE REVIEW

File: doc/motr-in-prose.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 _« Par ma foi ! il y a plus de quarante ans que je dis de la prose sans que j'en susse rien, et je vous suis le
 plus obligé du monde de m'avoir appris cela. »_
 
-This document attempts to be a master document for the whole of Motr, 
+This document attempts to be a primary document for the whole of Motr,
```
Comment (mukul-seagate11) [inline doc/motr-in-prose.md:6]: Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 1](https://app.codacy.com/gh/Seagate/cortx-motr/pullRequest?prid=9629868)
```suggestion
This document attempts to be a primary document for the whole of Motr,
```

File: doc/outdated/Containers.md [hunk 2]
```diff
@@ -10,8 +10,10 @@
...
-+ After a container is COW-ed no update should ever touch the blocks of the read-only master container. This is a necessary prerequisite of a scalable fsck implementation, that will achieve reasonable confidence in system consistency by background scan and check of periodically taken global snapshots.
-+ Migration. A container (either read-only or read-write) can migrate to another node. Typical scenarios of such migration are bulk re-integration of updates from a proxy-server to a set of master servers and moving snapshots.
++   After a container is COW-ed no update should ever touch the blocks of the read-only primary container. This is a necessary prerequisite of a scalable fsck implementation, that will achieve reasonable confidence in system consistency by background scan and check of periodically taken global snapshots.
+
++   Migration. A container (either read-only or read-write) can migrate to another node. Typical scenarios of such migration are bulk re-integration of updates from a proxy-server to a set of primary servers and moving snapshots.
```
Comment (mukul-seagate11) [inline doc/outdated/Containers.md:15]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/Seagate/cortx-motr/pullRequest?prid=9629868)
```suggestion
+ Migration. A container (either read-only or read-write) can migrate to another node. Typical scenarios of such migration are bulk re-integration of updates from a proxy-server to a set of primary servers and moving snapshots.
```

File: doc/outdated/Containers.md [hunk 3]
```diff
@@ -10,8 +10,10 @@
...
 + Snapshots. Containers are used for data and meta-data snap shotting. When a local snapshot is made as a part of object or system level snap shotting, a container involved into the snapshot is COW-ed, so that all updates are re-directed to a new container.
-+ After a container is COW-ed no update should ever touch the blocks of the read-only master container. This is a necessary prerequisite of a scalable fsck implementation, that will achieve reasonable confidence in system consistency by background scan and check of periodically taken global snapshots.
-+ Migration. A container (either read-only or read-write) can migrate to another node. Typical scenarios of such migration are bulk re-integration of updates from a proxy-server to a set of master servers and moving snapshots.
++   After a container is COW-ed no update should ever touch the blocks of the read-only primary container. This is a necessary prerequisite of a scalable fsck implementation, that will achieve reasonable confidence in system consistency by background scan and check of periodically taken global snapshots.
```
Comment (mukul-seagate11) [inline doc/outdated/Containers.md:13]: Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/Seagate/cortx-motr/pullRequest?prid=9629868)
```suggestion
+ After a container is COW-ed no update should ever touch the blocks of the read-only primary container. This is a necessary prerequisite of a scalable fsck implementation, that will achieve reasonable confidence in system consistency by background scan and check of periodically taken global snapshots.
```

**Comment by kanchan-chaudhari** [comment]
Not Required.

**Pull request closed without merging**